Purchases/Networking/RCBackend.m

```diff
@@ -434,7 +434,7 @@ - (void)postSubscriberAttributes:(RCSubscriberAttributeDict)subscriberAttributes
                        appUserID:(NSString *)appUserID
                       completion:(nullable void (^)(NSError *_Nullable error))completion {
     if (subscriberAttributes.count == 0) {
-        RCLog(@"called post subscriber attributes with an empty attributes dict!");
+        RCWarnLog(@"%@", RCStrings.attribution.empty_subscriber_attributes);
```

Comment:
why do some of these do `(@"%@", <string>); ` while others just pass it in directly?

Reply:
@aboedo The `(@"%@", <string>);` is to treat it as a string literal or else Xcode throws warnings. Others pass in directly have a string formatter and Xcode doesn't complain.